LayoutTests/fast/images/spatial-image-controls.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html><!-- webkit-test-runner [ SpatialImageControlsEnabled=true ] -->
+<script src="../../resources/ui-helper.js"></script>
+<style>
+    img {
+        width: 200px;
+        height: 200px;
+    }
+    div {
+        width: 200px;
+        position: relative;
+    }
+</style>
+<body>
+    <div>
+        <img controls src="resources/spatial.heic" id="spatial">
```

Comment:
We need to test what happens when the image is loaded lazily (loading=lazy):
https://html.spec.whatwg.org/multipage/urls-and-fetching.html#lazy-loading-attribute

Reply:
Note: Haven't added test yet but quick manual test shows it works (with network inspector to confirm image was in fact lazily loading)